components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAttemptsByType/src/IsAnalyticsAttemptsByType.jsx

```diff
@@ -414,15 +413,13 @@ class IsAnalyticsAttemptsByType extends Widget {
             theme = lightTheme;
         }
 
-        if (this.state.faultyProviderConf) {
+        if (this.state.isDataProviderConfigFault) {
             return (
                 <MuiThemeProvider theme={theme}>
                     <div style={divSpacings}>
-                        <Typography variant="title" gutterBottom align="center">
-                            {this.state.header}
-                        </Typography>
                         <Typography variant="body1" gutterBottom align="center">
-                            Unable to fetch data, please check the data provider configurations.
+                            Unable to fetch data from siddhi data provider,
```

Comment:
Lets fix casing as follows. 
`Unable to fetch data from Siddhi data provider. Please check the data provider configurations.`

Applies for other places as well.

Reply:
Fixed